src/main/java/uk/gov/hmcts/cmc/claimstore/events/ccj/CountyCourtJudgmentEvent.java

```diff
@@ -13,12 +14,13 @@
 
     private final Claim claim;
     private final String authorisation;
-    private final boolean issue;
+    private final CountyCourtJudgmentType countyCourtJudgmentType;
```

Comment:
We don't need this  property. The `Claim` object in this context has `CountyCourtJudgment` which contains type, and we should be using that in event listener methods: e.g.
```
claim.getCountyCourtJudgment().getCcjType()
```